rclpy/test/talker_nested.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+import time
+
+# this is needed to allow rclpy to be imported from the build folder
+sys.path.insert(0, os.getcwd())
+
+import rclpy
+from rclpy.qos import qos_profile_default
+
+
+def talker_nested():
+    rclpy.init([])
+
+    from geometry_msgs.msg import TransformStamped
```

Comment:
A TODO would be great. Hopefully the Python messages will get fixed soon and then this be moved.